### DISCUSSION TITLE
Integrating with react native

### DISCUSSION THREAD

**Discussion by xwiz** [qruto/laravel-wave#51] [Q&A]
Is it possible to use this with react native? I found [a library]( https://www.npmjs.com/package/react-native-sse) that should help but the implementations are different. Seems there's no particular pub-sub standard out there yet with respect to SSE.

**Comment by slavarazum** [comment]
I'm not very familiar with React Native, but it should accept JavaScript libraries and use them as they are. It uses the `fetch` request under the hood, so you don't need any additional libraries. Have you tried using it as is?

**Comment by slavarazum** [comment]
Please feel free to open an issue if you face any obstacles while integrating Wave with React Native.
